addons/mail/static/src/core/messaging_service.js

```diff
@@ -80,7 +80,7 @@ export class Messaging {
      * Import data received from init_messaging
      */
     initialize() {
-        this.rpc("/mail/init_messaging", {}, { silent: true }).then(
+        this.rpc("/mail/init_messaging", {context: this.env.services.user.context}, { silent: true }).then(
```

Comment:
(minor) lint: should apply automatic formatter
(should have spaces inside {}, probably line too long too)